src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -566,14 +573,36 @@ private JcrResourceBundle createResourceBundle(
      * </ol>
      */
     private Locale getParentLocale(Locale locale) {
```

Comment:
Can you please provide an explicit test coverage for this method? As you have a complete specification already in the comments, I think it should be easy to convert that into a test method.

(Changing the scope from private to protected should not be a problem, as this method is not exported publicly in OSGI.)

Reply:
Thank you for bringing this to my attention. I have added UT for this method separately.